Library/Homebrew/extend/ENV/super.rb

```diff
@@ -28,8 +28,7 @@ def self.extended(base)
   def self.bin
     return unless MacOS.has_apple_developer_tools?
 
-    bin = HOMEBREW_ENV_PATH.subdirs.reject { |d| d.basename.to_s > MacOS::Xcode.version }.max
-    bin.realpath unless bin.nil?
+    HOMEBREW_ENV_PATH.subdirs.max.realpath
```

Comment:
> Hence the correct refactoring of this for a generic OS would be to always return nil.

@UniqMartin I guess the question for this case is: is it superenv any more if it's not using any of the wrappers?